docs/transformations.md

```diff
@@ -0,0 +1,91 @@
+# Transformations {#transformations}
+<!--
+Copyright (C) The DDC development team, see COPYRIGHT.md file
+
+SPDX-License-Identifier: MIT
+-->
+
+On the top of DDC core - which provides the basic tools for the needs of performance-portable physical simulations
+(describing meshes with *DiscreteDomain* and representing physical fields with *Chunk*) - are build computation kernels
+to perform transformations. At the moment, the two available transformations are Discrete Fourier Transform and Spline methods.
+
+Those methods are performance portable (supported with CPU Serial, CPU OpenMP, CUDA and HIP backends). They are used
+in *examples/heat_equation_spectral.cpp* (for DFT) and *examples/characteristics_advection.cpp* (for splines).
+A particular use-case of splines is benchmarked in *benchmarks/splines.cpp*.
+
+## General presentation of transformations
+
+### Computation kernels dedicated to transformations in DDC
+
+A transformation is a change of basis, going from a function space to another. Because the available memory is finite,
+the basis are necessarily finite (which means it contains a finite set of basis functions, but those basis functions
+may be continuous).
+
+The use-cases of transformations for simulation or signal processing needs are plethoric: interpolation, spectral methods,
```

Comment:
![image](https://github.com/CExA-project/ddc/assets/25363473/8b12797d-ec19-4f0d-91be-46e28d591ffe)
"plethoric" is correct but archaic. I wasn't sure if it was actually a word. I suggest "multitude" or the more straightforward:
```suggestion
There are many use-cases of transformations for simulation or signal processing needs: interpolation, spectral methods,
```